dist/index.js

```diff
@@ -5827,9 +5827,9 @@ class EnableGithubAutomergeAction {
             core_1.debug(`Successfully retrieved merge-method as: ${mergeMethod}`);
             //
             // Step 2. Enable auto-merge.
-            core_1.debug(`Enabling auto-merge for pull-request #${pullRequest.number}`);
-            yield this.enableAutoMerge(pullRequestId, mergeMethod);
-            core_1.debug(`Successfully enabled auto-merge for pull-request #${pullRequest.number}`);
+            core_1.debug(`Enabling auto-merge for pull-request #${pullRequest.number}...`);
```

Comment:
Should this have been commited?

Reply:
Yep, this is re-generated automatically on each push.  Maybe a tad unintuitive... 😓